OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,306 @@
+Building OUD image with WebLogic Image Tool
```

Comment:
I strongly encourage you to get [`markserv`](https://www.npmjs.com/package/markserv) and use it to view your Markdown formatting. I would also encourage the use of `markdownlint` (or similar) tools to highlight common markdown issues.